ui/packages/app/web/src/components/Targets/utils.ts

```diff
@@ -0,0 +1,105 @@
+import moment from 'moment';
+
+export enum HealthStatus {
+  'Unspecified',
+  'Good',
+  'Bad',
+}
+
+export const getHealthStatus = (numericValue: number) => {
+  const label = HealthStatus[numericValue];
+  const colorVariants = {
+    Unspecified: 'neutral',
+    Good: 'success',
+    Bad: 'danger',
+  };
+  return {label, colorVariant: colorVariants[label]};
+};
+
+export const secondsInNanos = (seconds: number) => seconds * 1e9;
```

Comment:
I wonder if we can reuse this utility function [here](https://github.com/parca-dev/parca/blob/main/ui/packages/shared/functions/src/index.ts#L41) to format the values? 🤔

Reply:
The issue is that the formatting function `valueFormatter` currently only formats to a single unit (3.5 minutes vs. 3m 30s). I believe formatting a value to multiple units might be more understandable for larger values or durations. For example, it might be more understandable to see '6d 4h ago' instead of '6.17d ago'. What do you think @yomete?

Reply:
I modified the value formatter function in our @parca/functions file and created a separate function as well for this use case. In particular, I think a separate function is necessary if we want to communicate durations in a more human-readable manner - that is, instead of a single unit, communicating several (3m 30s instead of 3.5m)

Reply:
just seen it. love it!